e2e/Tests/iothub/device/FaultInjectionPoolAmqpTests.cs

```diff
@@ -13,5 +14,6 @@ namespace Microsoft.Azure.Devices.E2ETests
     public partial class FaultInjectionPoolAmqpTests : E2EMsTestBase
     {
         private static readonly string s_proxyServerAddress = TestConfiguration.IotHub.ProxyServerAddress;
+        private static readonly TimeSpan s_defaultOperationTimeout = TimeSpan.FromSeconds(30);
```

Comment:
Maybe put this field on E2EMsTestBase and have a method there to create a CTS with that value?

Reply:
Maybe with an override parameter for operation timeout?